compiler-core/src/language_server/engine.rs

```diff
@@ -791,9 +936,11 @@ where
 }
 
 fn type_completion(
-    module: Option<&EcoString>,
+    module: Option<&str>,
     name: &str,
     type_: &TypeConstructor,
+    insert_range: lsp::Range,
+    include_type_in_completion: bool,
```

Comment:
Let's always use an enum over a bool as it's not clear what `false` means outside of context